framework/packages/abstract-sdk/src/apis/ibc_memo/pfm.rs

```diff
@@ -0,0 +1,140 @@
+use std::collections::BTreeMap;
+
+use serde_cw_value::Value;
+
+use super::IbcMemoBuilder;
+
+/// Builder for [Packet Forward Middleware](https://github.com/cosmos/ibc-apps/tree/main/middleware/packet-forward-middleware) memos.
+pub struct PacketForwardMiddlewareBuilder {
+    receiver: String,
+    port: Option<String>,
+    hops: Vec<PacketForwardMiddlewareHop>,
+}
+
+impl PacketForwardMiddlewareBuilder {
+    /// Forward memo builder
+    pub fn new(receiver: impl Into<String>) -> Self {
```

Comment:
Move receiver to build fn and accept first hop in new function. 
```suggestion
    pub fn new(first_hop: Xyz) -> Self {}
    
    pub fn build(receiver: impl Into<String>) ... {}
```